src/main/java/com/team3/central/services/EventService.java

```diff
@@ -52,28 +58,36 @@ public com.team3.central.openapi.model.Event addEvent(String title, String name,
     return eventMapper.convertToModel(event);
   }
 
-  public Optional<com.team3.central.openapi.model.EventWithPlaces> getById(Long id) throws NotFoundException {
-    if(!eventRepository.existsById(id)) throw new NotFoundException("Index does not exist");
+  public Optional<com.team3.central.openapi.model.EventWithPlaces> getById(Long id)
+      throws NotFoundException {
+    if (!eventRepository.existsById(id)) {
+      throw new NotFoundException("Index does not exist");
+    }
     return eventRepository.findById(id).map(eventMapper::convertToEventWithPlaces);
   }
 
   public List<com.team3.central.openapi.model.Event> getAllEvents() {
-    return eventRepository.findAll()
-        .stream()
-        .map(eventMapper::convertToModel)
-        .collect(Collectors.toList());
+    List<com.team3.central.openapi.model.Event> list = new ArrayList<>();
+    for (Event event : eventRepository.findAll()) {
+      com.team3.central.openapi.model.Event convertToModel = eventMapper.convertToModel(event);
+      list.add(convertToModel);
+    }
+    return list;
```

Comment:
Why did you changed that ? I mean we still are using findAll() so this isn't better and it looks worse in my opinion. Could you revert ?